app/assets/javascripts/playspaces.js

```diff
@@ -0,0 +1,27 @@
+$(document).ready(function(){
+  var option = {
+    HK: ["Central", "North Point", "Repulse Bay", "The Peak", "Wan Chai", "Western"],
+    Kln: ["Ho Man Tin", "Tsim Sha Tsui", "Wong Tai Sin"],
+    NT: ["Sai Kung", "Sha Tin", "Tai Po"]
+  }
+
+  $("#playspace_region").on('change', function() {
+    $("#playspace_district").html("");
```

Comment:
Much as I would like to take sole credit for this, I got some help from Denis. This code was a pain!